### PR TITLE
Add `New Featured Artist: Haywyre` news post

### PR DESCRIPTION
**Pull request by pishifat**
- [x] linksys

### CODE REVIEW

File: news/2022-01-12-new-featured-artist-haywyre.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+date: 2022-01-12 09:00:00 +0000
+---
+
+A monument to contemporary dance music, **Haywyre** joins our Featured Artist library!
+
+![](https://assets.ppy.sh/artists/230/header.jpg)
+
+**Haywyre** is one of those musicians who feels almost too famous to really be part of our humble circle-clicking game.
+
+This grammy nominated producer and pianist has toured with the likes of *Zedd* and *GRiZ* thanks to the distinct vision he puts forth with each of his electronic tunes. Inspired by creators like our very own [KOAN Sound](https://osu.ppy.sh/beatmaps/artists/49) and various classical composers/pianists, **Haywyre**'s one-of-a-kind sound is surprisingly well suited for the world of rhythm gaming.
```
Comment (Walavouchey) [inline news/2022-01-12-new-featured-artist-haywyre.md:13]:
```suggestion
This grammy-nominated producer and pianist has toured with the likes of *Zedd* and *GRiZ* thanks to the distinct vision he puts forth with each of his electronic tunes. Inspired by creators like our very own [KOAN Sound](https://osu.ppy.sh/beatmaps/artists/49) and various classical composers/pianists, **Haywyre**'s one-of-a-kind sound is surprisingly well suited for the world of rhythm gaming.
```

**Comment by Walavouchey** [comment]
yes, more em dashes please. pretty sure that was a hyphen though

**Review by peppy — APPROVED** [review]

File: news/2022-01-12-new-featured-artist-haywyre.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/230/Two%20Fold%20Pt.%201/Haywyre%20-%20Dichotomy%20%28Soft%20Mix%29.mp3" type="audio/mpeg">
+</audio>
+
+---
+
+If you're sitting in front of your computer, [**Haywyre**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/230) is waiting for you to choose a song and open its pre-timed beatmap template in the editor. If you're reading this on your phone, [**Haywyre**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/230) encourages you to have a good day and stay hydrated. Yes, [**Haywyre**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/230) is sentient.
+
+Fun fact: **Haywyre** was internally used as a mockup example for Featured Artist pages many years ago! Fun fact \#2: you can be notified about our next Featured Artist as soon as it drops from the [@osugame Twitter](https://twitter.com/osugame).
```
Comment (ZeroPyrozen) [inline news/2022-01-12-new-featured-artist-haywyre.md:53]: 2 Fun facts in same line?

Comment (Ephemeralis) [reply news/2022-01-12-new-featured-artist-haywyre.md:53]: if he's daring, some day he'll do three!

this is fine otherwise, is a literary device

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**